src/olympia/search/tests/test_views.py

```diff
@@ -840,7 +840,7 @@ def test_results_name_query(self):
         # Try to match 'The Life Aquatic with SeaVan'.
         # We have prefix_length=4 so fuzziness matching starts
         # at the 4th character for performance reasons.
-        for term in ('life', 'aquatic', 'seavan', 'seav an'):
+        for term in ('life', 'aquatic', 'seavan', 'sea van'):
```

Comment:
So that's one of the maybe unfortunate? results of `match` queries analyzing queries. Imho that test isn't very realistic so I'd like to see how that performs in real life.